src/python/setup.py

```diff
@@ -135,8 +135,13 @@ def run(self):
         "requests >= 2.9.1",
         "scipy >= 1.1.0",
         "six >= 1.10.0",
-        "tensorflow >= 2.0.0",
     ]
+    if sys.platform == "darwin":
```

Comment:
Looks like a commit from your other PR made it into this one

Reply:
Good catch. Thanks. Fixed.